src/components/Header/header.css

```diff
@@ -0,0 +1,21 @@
+.website-header{
+    grid-area: header;
+    padding: 0.5em 1.5em;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    box-shadow: 0px 0px 10px 2px var(--primary-theme-shade-20);
+}
+
```

Comment:
In padding you can use px. Just a personal suggestion. in box-shadow you can use 0 instead of 0px